src/components/Date/DateRangePicker.tsx

```diff
@@ -29,7 +29,7 @@ export const DateRangePicker = ({
           value={from}
           onChange={(newValue) => {
             if (newValue) {
-              onFromChange(newValue);
+              onFromChange(newValue.startOf("day"));
```

Comment:
This will be start of the day _in the user's local timezone_. This is almost certainly not what was intended?  Can you check that this filtering is applied in the request made to ads-serve based on the ad server model of UTC days.

Reply:
Similarly with `endOf("day")` below